spring-xd-messagebus-kafka/src/main/java/org/springframework/xd/dirt/integration/kafka/KafkaMessageBus.java

```diff
@@ -550,7 +553,7 @@ protected Object handleRequestMessage(Message<?> requestMessage) {
 			try {
 				theRequestMessage = embeddedHeadersMessageConverter.extractHeaders((Message<byte[]>) requestMessage);
 			}
-			catch (UnsupportedEncodingException e) {
+			catch (Exception e) {
```

Comment:
Will the exception thrown also contain the message contents/headers that could not be extracted for logging?